test/terraform_linux_vm/module_vm/install_redhat.sh

```diff
@@ -0,0 +1,264 @@
+      sha256sum -c jdk-17_linux-x64_bin.rpm.sha256 || exit 1
+
+      sudo yum install -y jdk-17_linux-x64_bin.rpm
+    else
+      sudo yum install -y java-17-openjdk-devel
+    fi
+
+    java -version
+}
+
+install_tools(){
+    os=`cat /etc/redhat-release  | awk '{ print tolower($1) }'`
+    version=$(awk -F'=' '/VERSION_ID/{ gsub(/"/,""); print $2}' /etc/os-release | cut -d. -f1)
+    echo "Detect version: $os/$version"
+
+    if [[ "$os" == "centos" && "$version" -eq 8 ]]
+    then
+        echo "Update Centos Stream"
+        sudo sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+        sudo sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
+        sudo yum update -y
+    fi
+
+    if [[ "$version" -lt 8 ]]
+    then
+        echo "Install specific tools for RHEL < 8"
+        sudo yum install -y epel-release
+    fi
+
+    sudo yum install -y policycoreutils-python-utils
+}
+
+install_graviteeio_repository(){
+    echo "[graviteeio]
+name=graviteeio
+baseurl=https://packagecloud.io/graviteeio/rpms/el/7/\$basearch
+gpgcheck=0
+enabled=1
+gpgkey=https://packagecloud.io/graviteeio/rpms/gpgkey
+sslverify=1
+sslcacert=/etc/pki/tls/certs/ca-bundle.crt
+metadata_expire=300" | sudo tee /etc/yum.repos.d/graviteeio.repo > /dev/null
+    sudo yum -q makecache -y --disablerepo='*' --enablerepo='graviteeio'
+}
+
+open_ui_ports(){
+  ui_port=$(sudo semanage port -l | grep 8084 | wc -l)
+  if [[ "$ui_port" -eq 0 ]]
+  then
+      sudo semanage port -a -t http_port_t -p tcp 8084
+  else
+      sudo semanage port -m -t http_port_t -p tcp 8084
+  fi
+
+  portal_port=$(sudo semanage port -l | grep 8085 | wc -l)
+  if [[ "$portal_port" -eq 0 ]]
+  then
+      sudo semanage port -a -t http_port_t -p tcp 8085
+  else
+      sudo semanage port -m -t http_port_t -p tcp 8085
+  fi
+}
+
+get_current_public_ip() {
+  local public_ip
+
+  public_ip="$(curl -s -H Metadata:true --noproxy "*" "http://169.254.169.254/metadata/instance/network/interface?api-version=2021-02-01&format=json" | jq -r '.[].ipv4.ipAddress[].publicIpAddress')"
+  if echo -n "${public_ip}" | grep -q -E '^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$'
+  then
+    echo "${public_ip}"
+    return 0
+  fi
+
+  public_ip="$(curl -s 'https://ipv4.seeip.org')"
+  if echo -n "${public_ip}" | grep -q -E '^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$'
+  then
+    echo "${public_ip}"
+    return 0
+  fi
+
+  return 1
+}
+
+configure_frontend(){
+  local public_ip="$1"
+  if [[ -z "${public_ip}" ]]
+  then
+    echo "Missing IP argument." >&2
+    return 1
+  fi
+  sudo sed -i "/\"baseURL\": /{s#\"baseURL\": \".*\"#\"baseURL\": \"http://${public_ip}:8083/management\"#}" /opt/graviteeio/apim/management-ui/constants.json
+  sudo sed -i "/\"baseURL\": /{s#\"baseURL\": \".*\"#\"baseURL\": \"http://${public_ip}:8083/portal\"#}" /opt/graviteeio/apim/portal-ui/assets/config.json
+}
+
+install_graviteeio_from_repository(){
```

Comment:
maybe you should also implement upgrade? I had to manually run   sudo systemctl restart graviteeio-apim-gateway graviteeio-apim-rest-api to apply upgrade after I run install_graviteeio_from_repository with the new version. Not sure about UI though I only restarted BE services